Multisig/Presentation/UI Library/QRScanner/QRCodeScannerViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ScannerViewController.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+protocol QRCodeScannerViewControllerDelegate {
+    func scannerViewControllerDidScan(_ code: String)
+}
+
+class QRCodeScannerViewController: UIViewController {
+    @IBOutlet weak var headerLabel: UILabel!
+    var previewLayer: AVCaptureVideoPreviewLayer!
+    var captureSession: AVCaptureSession!
+    var delegate: QRCodeScannerViewControllerDelegate?
+    var header: String?
+    
+    enum Strings {
+        static let cameraAlertTitle = NSLocalizedString("camera_title", comment: "")
```

Comment:
My suggestion would be to use a localization key value in English as a comment, or to use some short description if localization key value is too long.